tests/test_dump_file.py

```diff
@@ -187,3 +187,71 @@ def test_copy_from(snap, tmp_path):
     assert numpy.all(read_snap.molecule == [0, 1, 2])
     assert read_snap.has_charge()
     assert numpy.allclose(read_snap.charge, [0, 1, -1])
+
+
+def test_faulty_dump_schema(snap, tmp_path):
+    # test image schema
+    f = open(tmp_path / "atoms.lammpstrj", "w")
+    f.write(
+        """
+    ITEM: TIMESTEP
+    0
+    ITEM: NUMBER OF ATOMS
+    1
+    ITEM: BOX BOUNDS pp pp pp
+    -7.0 7.0
+    -7.0 7.0
+    -7.0 7.0
+    ITEM: ATOMS id type ix iy
+    1 1 0 0 """
+    )
+    f.close()
+    filename = tmp_path / "atoms.lammpstrj"
+    with pytest.raises(OSError) as error:
+        pytest.raises(lammpsio.DumpFile(filename))
+
+    assert str(error.value) == "lammpsio requires 3-element vectors"
```

Comment:
I don't usually test the values of exceptions, since developers may change those. I just care that the right type is raised (here an OSError). Is there a reason you had to use the double raise check as well? I would think just this would be OK.
```suggestion
    with pytest.raises(OSError) as error:
        lammpsio.DumpFile(filename)
```

Reply:
I was double checking to see if the error value matched as well but what you said makes sense